packages/gatsby-plugin-referrer/.gitignore

```diff
@@ -0,0 +1,2 @@
+/*.js
+!index.js
```

Comment:
I think we can remove this because `index.js` is not no-op file (like in most gatsby plugins) and it would actually be built from `src/index.js` - this would also mean not commiting `packages/gatsby-plugin-referrer/index.js` into repository